homeassistant/components/assist_satellite/entity.py

```diff
@@ -46,6 +46,9 @@ class AssistSatelliteEntity(entity.Entity):
     entity_description: AssistSatelliteEntityDescription
     _attr_should_poll = False
     _attr_state: AssistSatelliteState | None = None
+    _attr_supported_features: AssistSatelliteEntityFeature = (
+        AssistSatelliteEntityFeature(0)
+    )
```

Comment:
Do we have to specify type if we do assignment?
```suggestion
    _attr_supported_features AssistSatelliteEntityFeature(0)
```

Reply:
I guess not